### PR TITLE
src: Index is a 2D vector of int and number of cells is an unsigned

### PR DESCRIPTION
**Pull request by jhidalgocarrio**
It attempts to fixed the issue raised by @jmachowinski regarding Index.

### CODE REVIEW

File: src/grid/GridMap.hpp [hunk 1]
```diff
@@ -380,11 +380,11 @@ namespace maps { namespace grid
...
 
-        void extend(const Index &minSize)
+        void extend(const Vector2ui &minSize)
         {
-            Index newSize;
-            newSize.x() = std::max(minSize.x(), getNumCells().x());
-            newSize.y() = std::max(minSize.y(), getNumCells().y());
+            Vector2ui newSize;
+            newSize.x() = static_cast<unsigned int>(std::max(minSize.x(), getNumCells().x()));
+            newSize.y() = static_cast<unsigned int>(std::max(minSize.y(), getNumCells().y()));
```
Comment (jmachowinski) [inline src/grid/GridMap.hpp]: you should not need the static cast here

File: src/grid/ElevationMap.cpp [hunk 2]
```diff
@@ -74,7 +74,7 @@ float ElevationMap::getMeanElevation(const Vector3d& pos) const
     if (at(idx) == ELEVATION_DEFAULT)
         return ELEVATION_DEFAULT;
 
-    Vector3d pos_t;
+    Vector3d pos_t(0.00, 0.00, 0.00);
```
Comment (jmachowinski) [inline src/grid/ElevationMap.cpp:77]: no need to init here, fromGrid will initialize it

**Comment by jmachowinski** [comment]
apart from my two comments, it looks fine to me

File: src/grid/GridMap.hpp (hunk 1 above)
Comment (jhidalgocarrio) [inline src/grid/GridMap.hpp]: Right! Changed

File: src/grid/ElevationMap.cpp (hunk 2 above)
Comment (jhidalgocarrio) [inline src/grid/ElevationMap.cpp:77]: This one removes a compilation warning

**Pull request merged**